operator/model/src/main/java/io/apicurio/registry/operator/api/v3/status/Conditions.java

```diff
@@ -0,0 +1,160 @@
+        return lastTransitionTime;
+    }
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss[XXX][VV]")
+    public void setLastTransitionTime(ZonedDateTime lastTransitionTime) {
+        this.lastTransitionTime = lastTransitionTime;
+    }
+
+    /**
+     * message is a human readable message indicating details about the transition. This may be an empty
+     * string.
+     */
+    @JsonProperty("message")
+    @Required()
+    @JsonPropertyDescription("message is a human readable message indicating details about the transition. This may be an empty string.")
+    @JsonSetter(nulls = Nulls.SKIP)
+    private String message;
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    /**
+     * observedGeneration represents the .metadata.generation that the condition was set based upon. For
+     * instance, if .metadata.generation is currently 12, but the .status.conditions[x].observedGeneration is
+     * 9, the condition is out of date with respect to the current state of the instance.
+     */
+    @JsonProperty("observedGeneration")
+    @Min(0.0)
+    @JsonPropertyDescription("observedGeneration represents the .metadata.generation that the condition was set based upon. For instance, if .metadata.generation is currently 12, but the .status.conditions[x].observedGeneration is 9, the condition is out of date with respect to the current state of the instance.")
+    @JsonSetter(nulls = com.fasterxml.jackson.annotation.Nulls.SKIP)
+    private Long observedGeneration;
+
+    public Long getObservedGeneration() {
+        return observedGeneration;
+    }
+
+    public void setObservedGeneration(Long observedGeneration) {
+        this.observedGeneration = observedGeneration;
+    }
+
+    /**
+     * reason contains a programmatic identifier indicating the reason for the condition's last transition.
+     * Producers of specific condition types may define expected values and meanings for this field, and
+     * whether the values are considered a guaranteed API. The value should be a CamelCase string. This field
+     * may not be empty.
+     */
+    @JsonProperty("reason")
+    @Required()
+    @Pattern("^[A-Za-z]([A-Za-z0-9_,:]*[A-Za-z0-9_])?$")
+    @JsonPropertyDescription("reason contains a programmatic identifier indicating the reason for the condition's last transition. Producers of specific condition types may define expected values and meanings for this field, and whether the values are considered a guaranteed API. The value should be a CamelCase string. This field may not be empty.")
+    @JsonSetter(nulls = com.fasterxml.jackson.annotation.Nulls.SKIP)
+    private String reason;
+
+    public String getReason() {
+        return reason;
+    }
+
+    public void setReason(String reason) {
+        this.reason = reason;
+    }
+
+    public enum Status {
```

Comment:
Move these to a separate class please for better readability.

Reply:
ok